### PR TITLE
I7495 submission listing additions

### PR DESCRIPTION
**Pull request by jardakotesovec**

### CODE REVIEW

File: classes/submission/maps/Schema.php [hunk 1]
```diff
@@ -445,6 +445,7 @@ protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, b
                 'roundId' => (int) $reviewAssignment->getReviewRoundId(),
                 'recommendation' => $reviewAssignment->getRecommendation(),
                 'dateCancelled' => $reviewAssignment->getData('dateCancelled'),
+                'reviewerId' => $anonymizeReviews && $anonymizeReviews->contains($reviewAssignment->getId()) ? null : $reviewAssignment->getId(),
                 'reviewerFullName' => $anonymizeReviews && $anonymizeReviews->contains($reviewAssignment->getId()) ? '' : $reviewAssignment->getData('reviewerFullName'),
```
Comment (jardakotesovec) [inline classes/submission/maps/Schema.php:449]: I am using submission.reviewAssignments to list the reviewers. And when adding actions I also needed reviewerId, to be able to "Login as".

**Comment by jardakotesovec** [comment]
@Vitaliy-1 Could you please have a quick look? Very small PR.

File: classes/submission/maps/Schema.php (hunk 1 above)
Comment (Vitaliy-1) [reply classes/submission/maps/Schema.php:449]: `'reviewerId' => ... $reviewAssignment->getReviewerId()`?

**Review by Vitaliy-1 — CHANGES_REQUESTED** [review]
Thanks! Requires one small tweak

**Comment by jardakotesovec** [comment]
@Vitaliy-1 Small, yet quite important one :-). Thank you. Merging now.

**Pull request merged**